KoodooHQ/bit-spike
# fix: resolve tag error

## Description

Bit pipeline tagging is failing because previously exported props are no longer exported

### Summary

I have removed the lines where the exported props are declared.

### Commitment to review

5 minutes to review the files

### Review instructions

- See the build failure on main is due to props on the comparison header no longer existing due to recent refactor. 
- Confirm the references are removed.

### Acceptance criteria

  - [ ] No more references to comparison header props
